src/index.js

```diff
@@ -0,0 +1,27 @@
+import './style.css';
+import {
+  loadTasks,
+  addTask,
+  deleteTaks,
+  updateTaks,
+  deleteAllTaks,
+} from './modules/functions.js';
+import changeStatus from './modules/statusUpdate.js';
+
+const ul = document.querySelector('.ul');
+loadTasks(localStorage.getItem('lists'), ul);
+const addInput = document.querySelector('.add-input form');
+addInput.addEventListener('submit', (event) => {
+  event.preventDefault();
+  addTask(document.querySelector('#new-item'), ul);
+  deleteTaks(ul);
```

Comment:
Pay attention to typos in your code; It can lead to some debugging challenges because you may miss later in your code that your function name was misspelled and then try to call the function or the variable with the correct spelling and not understand why the code is not working 😵‍💫